news/2021-07-26-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,133 @@
+- The bonus also has a timing-based component not shown above. On the plot above it is assumed that the timing factor is minimal (and so the angle bonus is the biggest it can be)
+
+---
+
+The changes listed above will all go into effect **immediately**. This is important to note as the next batch of new changes are more aimed at playing a bigger role in future developments, and may not be of that much interest to the average player at the moment.
+
+If you're just after the juicy details on what affects you today, you can safely skip the rest of this article. If you're interested in some formative groundwork for the future and have the skills to implement and fiddle with them in code, read on.
+
+Note: the following changes only affect osu!lazer and **star rating** for now.
+
+### osu!mania: Improved accuracy of lazer difficulty calculation
+
+osu!team's very own [**smoogipoo**](https://osu.ppy.sh/users/1040328) has [successfully managed](https://github.com/ppy/osu/pull/10453) to port the mania difficulty calculation algorithm from stable.
+
+While not 100% perfect, it is essentially as close as it can get, resulting in differences below 0.01 stars for the vast majority of all cases. Only one mania-specific map and seven converts with leaderboards have any changes in star rating above 0.2 stars, and most are due to rounding shenanigans and unintended behaviour in stable (commonly referred to as gimmicks) which is fixed in lazer.
+
+Though solely in the domain of osu!lazer, it may eventually resolve a [fair](https://github.com/ppy/osu-web/issues/7846) [few](https://github.com/ppy/osu-web/issues/6075) [issues](https://github.com/ppy/osu-web/issues/5442) with displaying mania map difficulties on the website, by reusing lazer difficulty calculation components in other areas of the osu! infrastructure.
+
+### All rulesets: More flexibility in the structure of skills
+
+As community members explore new ideas in the domain of difficulty and performance calculation, the limitations of how things were previously set up can come to bear in quite stifling ways. 
+
+Long-term contributor [**Syrin**](https://osu.ppy.sh/users/5701575) has made great strides in revamping this process for the benefit of anyone interested in creating their own rulesets in the changes listed below:
+
+- [Skills now receive a list of active mods](https://github.com/ppy/osu/pull/11687), which will allow their creators to very easily create mod-specific adjustments in the future. Notably, this new capability has already been utilized to [clean up existing osu!catch star rating calculations](https://github.com/ppy/osu/pull/11857).
+- Skills were previously hard limited in how many objects they could operate on. [A new change](https://github.com/ppy/osu/pull/11688) allows rules to now manually specify directly how many objects they can act on in relation to any one object, allowing for much more robust algorithms to be developed.
```

Comment:
in that case it would be

```suggestion
- Skills were previously hard-limited in how many objects they could operate on. [A new change](https://github.com/ppy/osu/pull/11688) now allows direct and manual specification of how many objects can be acted on in relation to any one object, allowing for much more robust algorithms to be developed.
```